### DISCUSSION TITLE
Local Authentication Setting

### DISCUSSION THREAD

**Discussion by pillface** [simple-login/app#1453] [Q&A] [answered]
I'm having problems with the Local Authentication in Settings not staying on.

I have setup a Security Key on the web site and it works but the app seems to ignore this. The app still only uses the Authy 2FA. Setting Local Authentication in the settings on the app to "Restrict unwanted access to your SimpleLogin account on this device with biometric authentication of device passcode" is ignored when I log out/in. Furthermore this setting is not staying on.

Am I missing something?

**Comment by ntnhon** [comment, marked as answer]
Hey, that's the expected behavior. That's why it's "local". Whenever you're logged out, the app will reset this settings and you have to explicitly enable it.
